seeding/seed.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+#Get absolute path to seeding dir
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cat $SCRIPT_DIR/seed.sql | docker exec -i SHOW-database /bin/bash -c 'psql -U postgres -d show_db'
+
+echo "Database seeding complete"
```

Comment:
can you display a error message if the above process fails?

Reply:
you'll know if it fails from the sql logs 

also the toast msg won't ever appear